Expensify/App
# [N6.1] Use Google AddressSearch component in VBA Company Step

### Details


### Fixed Issues

Half of https://github.com/Expensify/Expensify/issues/179233

### Tests

1. Head on over to new.expensify.com.dev or https://staging.new.expensify.com/
2. Make a new account
3. Validate that account
    a) If you're on staging, check your email for the validation link
    b) If you're on dev, run `Expensidev/script/clitools.sh validator:account` to validate the new account
4. The Create Menu should pop up at the lower left, but if it doesn't click the green plus button down there
5. Click 'New Workspace' (then click 'Expensify Card' if you're on mobile)
6. Click 'Get Started' in the modal that pops up
7. Click 'Connect Manually' in the VBA modal that comes up on the right hand side
8. Use the following info for the first step
    a) Routing number 011401533 
    b) Account number 1111222233331111
9. Click 'Save & Continue'
10. Type `42` in the 'Company Address' field
11. Wait a sec <--- might want to look into this. The response time can be long.
12. Verify that the address suggestions appear. 
13. Click one of them and verify that it enters that data into the 'Company Address' field

### Tested On

- [x] Mobile Web <--- finicky at best

### Screenshots


#### Web


<img width="663" alt="Screen Shot 2021-10-04 at 2 21 09 PM" src="https://user-images.githubusercontent.com/49007721/135919665-9144c9be-6a44-4dc4-b3e3-b06aa63b1964.png"> 

<img width="663" alt="Screen Shot 2021-10-04 at 2 24 38 PM" src="https://user-images.githubusercontent.com/49007721/135919718-b87132ee-3838-4121-b3e5-c8936840aa9f.png">

#### Mobile 

Web



<img width="560" alt="Screen Shot 2021-10-04 at 2 31 28 PM" src="https://user-images.githubusercontent.com/49007721/135920585-7cd6a768-ba9b-46dd-9ac1-a546bf8dca3d.png">

#### Desktop


<img width="612" alt="Screen Shot 2021-10-04 at 2 34 07 PM" src="https://user-images.githubusercontent.com/49007721/135921007-0a77c637-e367-49d1-8c81-17e449be2e7c.png">

<img width="612" alt="Screen Shot 2021-10-04 at 2 34 20 PM" src="https://user-images.githubusercontent.com/49007721/135921027-f9f42344-13ff-4d69-99c1-4e9e50aa6ff4.png">

#### iOS


<img width="560" alt="Screen Shot 2021-10-04 at 3 00 00 PM" src="https://user-images.githubusercontent.com/49007721/135924234-6f563220-95b5-4602-a2ed-af36b4acc628.png">

<img width="560" alt="Screen Shot 2021-10-04 at 3 00 08 PM" src="https://user-images.githubusercontent.com/49007721/135924241-dabba4f3-2f19-45da-ba53-3bf439485aae.png">

We are getting an error here that we should not use VirtualizedLists inside of a scroll view (the component provided by this new npm package uses a VirtualizedList). Here is the issue that was opened in the repo of the npm package. Looking into potential solutions for this. https://github.com/FaridSafi/react-native-google-places-autocomplete/issues/475#issuecomment-930318613

#### Android

<img width="608" alt="Screen Shot 2021-10-04 at 5 49 26 PM" src="https://user-images.githubusercontent.com/49007721/135939373-5d9ea463-9c6b-451f-8bcf-182d6cbf70f0.png">
<img width="608" alt="Screen Shot 2021-10-04 at 5 49 37 PM" src="https://user-images.githubusercontent.com/49007721/135939378-39dd2d02-83a6-4471-b7ba-762faf06792f.png">
